thothDiscussion.md

```diff
@@ -0,0 +1,37 @@
+ 
+![Goals - Stephen's original slide](thothGoals.png)
+
+
+
+[from Stephen’s original slides =>GOALS Agreed!! 
+Assumption: novices have reasonable coding skills in their chosen language(s) - or there are numerous mature, free resources in order to be able to attain coding skills – so focus need not be on the coding aspect.
+
+However, the peripheral skills needed to transition into the workplace might not be known.  The requisite skills need to be 
+
+* Identified
+* Prioritised
+* Practiced 
+
+
+And this is the problem that Thoth aims to solve!
+Provide novices with a narrow shortlist of peripheral skills, prioritised, along with a framework of resources/support to gain practical experience. The aim to help them to get to the workplace faster and with confidence 
+
+[Objectives]
+1. Thoth aims to provide a mentee looking for a specific job role (e.g. Front-End web-developer] with a validated roadmap of prioritised skills split by  sub-domain 
+
+![Peripheral Skills Map example](thothmap.png)
```

Comment:
This is a correct use of an image as that same layout is impossible / difficult to achieve with markdown